### DISCUSSION TITLE
Excluded folders with subfolder content

### DISCUSSION THREAD

**Discussion by TfTHacker** [brianpetro/obsidian-smart-connections#177] [Q&A]
I used the folder_exclusions in settings, but while tracing the activity of the plugin I saw file contents from the exclusion folders being sent up. This is the value I gave to this field in settings:

00-Folders/10-TfT, 00-Folders/zArchive, Areas/Entities

So three folders and they all have subfolders with files. Should these be defined differently? 

Also, the path_only variable isn't clear to me what it's doing.

**Comment by TfTHacker** [comment]
I should probably move this over to a bug unless I am using the UI wrong. This is a rather big deal that excluded content would be getting sent to the AI.

Thoughts?

**Reply by brianpetro** [reply]
@TfTHacker thanks for letting me know about this!

Moving that same value in the path_only setting might work. 

If you're on Windows, the current setting might need to replace '/' to '\\'.

Please do make an issue, it helps a lot. This might be better named an improvement, so feature request, specifically for the settings UI/UX. 

Updating the settings is pretty high priority already, but getting some specific improvement ideas would be really helpful. 

This link has some more notes on the settings https://github.com/brianpetro/obsidian-smart-connections#settings

Thanks,
Brian 🌴
